Rest api first draft

billy-the-fish opened the pull request.

billy-the-fish committed:
chore: add first files.

billy-the-fish committed:
chore: first draft of the REST API docs

josesahad commented on getting-started/get-started-devops-as-code.md:
```diff
@@ -0,0 +1,196 @@
+
+## Prerequisites
+
+<IntegrationPrereqs />
```
this is not a requirement

josesahad commented on getting-started/get-started-devops-as-code.md:
```diff
@@ -0,0 +1,196 @@
+   {
+     "name": "my-first-service",
+     "service_type": "TIMESCALEDB",
+     "region_code": "google-europe-west1",
```
this is wrong.. I can check the API spec, but let's use "us-east-1"

billy-the-fish committed:
chore: update and test.

billy-the-fish committed:
chore: update and test.

billy-the-fish committed:
Merge branch 'latest' into rest-api

billy-the-fish committed:
Merge branch 'latest' into rest-api

billy-the-fish committed:
Merge branch 'latest' into rest-api

josesahad commented on api/api-reference.md:
```diff
@@ -0,0 +1,838 @@
+**API Version:** 1.0.0  
+**Base URL:** `https://api.tigerdata.com/public/v1`  
+**Development URL:** `https://console.dev.timescale.com/public/api/v1`  
+**Local Development URL:** `http://localhost:8080/public/api/v1`
```
Let's remove these two

josesahad commented on api/api-reference.md:
```diff
@@ -0,0 +1,838 @@
+## Overview
+
+**API Version:** 1.0.0  
+**Base URL:** `https://api.tigerdata.com/public/v1`  
```
https://console.cloud.timescale.com/public/api/v1

josesahad committed:
Fix spec. Better examples and documentation

josesahad committed:
Fix replica set retrieval docs

billy-the-fish committed:
Merge branch 'latest' into rest-api

Signed-off-by: Iain Cox <iain@timescale.com>

billy-the-fish committed:
chore: few small updates.

billy-the-fish commented:
I've updated to remove the healthcheck that does not seem to work, and changed the workflow a bit. A few small changes to the API ref (Make each endpoint an action) and we are good to go.

billy-the-fish committed:
chore: few small updates.

billy-the-fish committed:
Merge branch 'latest' into rest-api

billy-the-fish committed:
Merge branch 'latest' into rest-api

billy-the-fish committed:
chore: change title to give space for the CLI

billy-the-fish committed:
Merge branch 'latest' into rest-api

billy-the-fish committed:
Merge branch 'latest' into rest-api

billy-the-fish committed:
Merge branch 'latest' into rest-api

billy-the-fish committed:
chore: updates on review.

billy-the-fish committed:
Merge branch 'rest-api' of https://github.com/timescale/docs into rest-api

billy-the-fish commented on api/api-reference.md:
```diff
@@ -0,0 +1,849 @@
+
+## Service Management
+
+You use this endpoint to create and manage the following Tiger Posgres services:
```
@josesahad , are you ok with me updating mentions of Tiger Postgres to say Tiger service in this file, or would you prefer to fix in source and regenerate?

billy-the-fish committed:
Merge branch 'latest' into rest-api
